src/plugin/tasks/serverless/snippets/serverless/test/sls-cf-stack-routes.yml

```diff
@@ -0,0 +1 @@
+route: true
```

Comment:
With format changed from `txt` to `yml`. This file would error if it was empty. I put a dummy value here.